v8/config/krb5conf.go

```diff
@@ -726,3 +731,100 @@ func (c *Config) JSON() (string, error) {
 	}
 	return string(b), nil
 }
+
+var expandMap map[string]string
+
+func init() {
```

Comment:
I like to avoid using Go's init function as if there are issues with it during runtime it can be confusing to debug as it was not explicitly called by another function. Could we initiate the expandMap within the LibDefaults.parseLines() method?